### PR TITLE
update to v0.4.0 version

### PR DESCRIPTION
**Pull request by oxxenix**

### CODE REVIEW

**Comment by oxxenix** [comment]
@byako, @eero-t, PTAL

File: charts/intel-gpu-resource-driver/README.md [hunk 1]
```diff
@@ -34,10 +35,12 @@ helm show values intel/intel-gpu-resource-driver
...
-|parameter| value |
-|---------|-----------|
-| `image.repository` | `intel` |
-| `image.tag` | `v0.3.0` |
+| Key | Type | Default |
+|-----|------|---------|
+| image.repository | string | `intel` |
+| image.name | string | `"IfNotPresent"` |
+| image.pullPolicy | string | `"intel-gpu-resource-driver"` |
+| image.tag | string | `""` |
```
Comment (eero-t) [inline charts/intel-gpu-resource-driver/README.md:43]: Do not seem to be valid image name, pull policy or tag.

File: charts/intel-gpu-resource-driver/README.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
 # Dynamic Resource Allocation (DRA) Intel GPU Driver Helm Chart
+![Version: 0.4.0](https://img.shields.io/badge/Version-0.4.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v0.4.0](https://img.shields.io/badge/AppVersion-v0.4.0-informational?style=flat-square)
```
Comment (eero-t) [inline charts/intel-gpu-resource-driver/README.md:2]: While badges seem popular, do we want image links to external 3rd party sites?

Only <10% of the current `README.md` files in this repo have them, and none in the top level chart directories:
```
$ git grep -l shields
charts/evi-hbase-0.8.3/charts/hbase/README.md
charts/evi-hbase-2.1.0/charts/hbase/README.md
charts/evi-minio-0.8.0/charts/minio/README.md
charts/evi-minio-0.8.1/charts/minio/README.md
charts/evi-minio-0.8.4/charts/minio/README.md
charts/evi-minio-2.1.0/charts/minio/README.md
charts/new-hbase-0.8.0/charts/hbase/README.md
```

**Review by eero-t — CHANGES_REQUESTED** [review]
Other changes look OK (match driver changes), but values table defaults in README are  messed up.

File: charts/intel-gpu-resource-driver/README.md (hunk 1 above)
Comment (oxxenix) [reply charts/intel-gpu-resource-driver/README.md]: should I put there the tag v0.4.0 or is it ok to leave empty string?

File: charts/intel-gpu-resource-driver/README.md (hunk 2 above)
Comment (oxxenix) [reply charts/intel-gpu-resource-driver/README.md:2]: I am okay with both options, either have them or not

File: charts/intel-gpu-resource-driver/README.md (hunk 1 above)
Comment (eero-t) [reply charts/intel-gpu-resource-driver/README.md]: Default **is** v0.4.0, isn't it?

Comment (oxxenix) [reply charts/intel-gpu-resource-driver/README.md]: yes, I will update

File: charts/intel-gpu-resource-driver/README.md (hunk 2 above)
Comment (oxxenix) [reply charts/intel-gpu-resource-driver/README.md]: removed

**Pull request merged**